TECHNICAL_VALUES_AND_PRIORITIZATION.md

```diff
@@ -0,0 +1,65 @@
+# Technical Values and Priorities
```

Comment:
```suggestion
# Technical values and priorities
```

Reply:
titles should be title case right?

Reply:
It depends. Titles of books and white papers? Yes. Titles of chapters of books or web articles? At least according to some style guides, those should be sentence case. I think of the examples at https://docs.microsoft.com/en-us/style-guide/text-formatting/formatting-titles, this most closely fits with a web article, so sentence case.